LibGit2Sharp.Tests/TestHelpers/Constants.cs

```diff
@@ -51,6 +51,11 @@ public static bool IsRunningOnUnix
             return null;
         }
 
+        public static bool PrivateRepoCertificateCheck(Certificate certificate, bool valid, string host)
```

Comment:
If its just returning true then why do we need it is there a future change that I should be aware of?

Reply:
No, it's basically just the same as a fake handler right above it 🙂